packages/codemods/src/transforms/popper.ts

```diff
@@ -0,0 +1,21 @@
+import { API, FileInfo } from 'jscodeshift';
+import { getImportInfo, renameProp } from '../codemod-helpers';
+import { JSCodeShiftOptions } from '../types';
+
+export const parser = 'tsx';
+
+const RENAME_MAP = {
+  offsetDistance: 'offsetByMainAxis',
+  offsetSkidding: 'offsetByCrossAxis',
+};
```

Comment:
```suggestion
  renderContent: 'children',
  arrowClassName: 'arrowProps', // => arrowProps={{ iconClassName: "" }}
  forcePortal: 'usePortal',
  portalRoot: 'usePortal',
};
```

при этом исключить одновременно использование `forcePortal` и `portalRoot`.